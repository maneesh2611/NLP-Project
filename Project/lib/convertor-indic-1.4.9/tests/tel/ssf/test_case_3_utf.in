<Sentence id="1">			
1	((	NP	<fs af='కమిటీ,n,n,sg,,d,0,0' head="కమిటీ">
1.1	కమిటీ	NN	<fs af='కమిటీ,n,n,sg,,d,0,0' name="కమిటీ">
	))		
2	((	NP	<fs af='కార్యనిర్వహణ,n,n,sg,,,కి,ki' head="కార్యనిర్వహణకు">
2.1	కార్యనిర్వహణకు	NN	<fs af='కార్యనిర్వహణ,n,n,sg,,,కి,ki' name="కార్యనిర్వహణకు">
	))		
3	((	NP	<fs af='మౌలికం,n,n,sg,,o,టి,ti' head="మౌలిక">
3.1	అవసరమైన	JJ	<fs af='అవసరమవ్వు,v,any,any,any,,ఇన,ina' poslcat="NM">
3.2	మౌలిక	NN	<fs af='మౌలికం,n,n,sg,,o,టి,ti' name="మౌలిక">
	))		
4	((	NP	<fs af='ఏర్పాటు,n,n,pl,,,పై,pE' head="ఏర్పాట్లపై">
4.1	ఏర్పాట్లపై	NN	<fs af='ఏర్పాటు,n,n,pl,,,పై,pE' name="ఏర్పాట్లపై">
	))		
5	((	NP	<fs af='పర్యటన,n,n,sg,,,లో,lo' head="పర్యటనలో">
5.1	ఈ	DEM	<fs af='ఈ,avy,,,,,,' poslcat="NM">
5.2	పర్యటనలో	NN	<fs af='పర్యటన,n,n,sg,,,లో,lo' name="పర్యటనలో">
	))		
6	((	RBP	<fs af='భాగం,n,,sg,,,లో,gA' head="భాగంగా"  poslcat="NM">
6.1	భాగంగా	RB	<fs af='భాగం,n,,sg,,,లో,gA' name="భాగంగా"  poslcat="NM">
	))		
7	((	NP	<fs af='తాము,pn,n,sg,2,d,0,0' head="తాము">
7.1	తాము	PRP	<fs af='తాము,pn,n,sg,2,d,0,0' name="తాము">
	))		
8	((	VGF	<fs af='సమీక్షించుకుంటామన్నాఉన్క్,v,,,,,,' head="సమీక్షించుకుంటామన్నారు"  poslcat="NM">
8.1	సమీక్షించుకుంటామన్నారు	VM	<fs af='సమీక్షించుకుంటామన్నాఉన్క్,v,,,,,,' name="సమీక్షించుకుంటామన్నారు"  poslcat="NM">
8.2	।	SYM	<fs af='।,punc,,,,,,'>
	))		
</Sentence>			
