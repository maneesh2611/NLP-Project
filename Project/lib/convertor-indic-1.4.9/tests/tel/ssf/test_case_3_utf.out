<Sentence id="1">			
1	((	NP	<fs af='kamitI,n,n,sg,,d,0,0' head="kamitI">
1.1	kamitI	NN	<fs af='kamitI,n,n,sg,,d,0,0' name="kamitI">
	))		
2	((	NP	<fs af='kAryanirvahaNa,n,n,sg,,,ki,ki' head="kAryanirvahaNaku">
2.1	kAryanirvahaNaku	NN	<fs af='kAryanirvahaNa,n,n,sg,,,ki,ki' name="kAryanirvahaNaku">
	))		
3	((	NP	<fs af='mOlikaM,n,n,sg,,o,ti,ti' head="mOlika">
3.1	avasaramEna	JJ	<fs af='avasaramavvu,v,any,any,any,,ina,ina' poslcat="NM">
3.2	mOlika	NN	<fs af='mOlikaM,n,n,sg,,o,ti,ti' name="mOlika">
	))		
4	((	NP	<fs af='erpAtu,n,n,pl,,,pE,pE' head="erpAtlapE">
4.1	erpAtlapE	NN	<fs af='erpAtu,n,n,pl,,,pE,pE' name="erpAtlapE">
	))		
5	((	NP	<fs af='paryatana,n,n,sg,,,lo,lo' head="paryatanalo">
5.1	I	DEM	<fs af='I,avy,,,,,,' poslcat="NM">
5.2	paryatanalo	NN	<fs af='paryatana,n,n,sg,,,lo,lo' name="paryatanalo">
	))		
6	((	RBP	<fs af='BAgaM,n,,sg,,,lo,gA' head="BAgaMgA"  poslcat="NM">
6.1	BAgaMgA	RB	<fs af='BAgaM,n,,sg,,,lo,gA' name="BAgaMgA"  poslcat="NM">
	))		
7	((	NP	<fs af='wAmu,pn,n,sg,2,d,0,0' head="wAmu">
7.1	wAmu	PRP	<fs af='wAmu,pn,n,sg,2,d,0,0' name="wAmu">
	))		
8	((	VGF	<fs af='samIkRiMcukuMtAmannAunk,v,,,,,,' head="samIkRiMcukuMtAmannAru"  poslcat="NM">
8.1	samIkRiMcukuMtAmannAru	VM	<fs af='samIkRiMcukuMtAmannAunk,v,,,,,,' name="samIkRiMcukuMtAmannAru"  poslcat="NM">
8.2	ред	SYM	<fs af='ред,punc,,,,,,'>
	))		
</Sentence>			
